terraform/lambdafunction.tf

```diff
@@ -50,8 +50,8 @@ resource "aws_lambda_function" "SyncApplicants" {
 # Scheduler to sync master tables
 resource "aws_cloudwatch_event_rule" "hmbc_to_ien_masters" {
   name                = local.sync_master_data_scheduler
-  description         = "7:00AM UTC - 12:00AM PST"
-  schedule_expression = "cron(0 7 * * ? *)"
+  description         = "7:00AM UTC - 12:00AM PST On the 15th of every month"
```

Comment:
It would be great if this is recorded on README.

Reply:
I'll update it, though we will need to change this back once we get the new api.